docs/data_entry_flow_index.md

```diff
@@ -129,6 +139,77 @@ class ExampleConfigFlow(data_entry_flow.FlowHandler):
         return self.async_show_form(step_id="init", data_schema=vol.Schema(data_schema))
 ```
 
+#### Labels & Descriptions
+
+Translations for the form are added to `strings.json` in a key for the `step_id`. That object may contain the folowing keys:
+
+|        Key         |       Value        | Notes                                                                                                                                        |
+| :----------------: | :----------------: | :------------------------------------------------------------------------------------------------------------------------------------------- |
+|      `title`       |    Form heading    | Do not include your brand name. It will be automatically injected from your manifest.                                                        |
+|   `description`    | Form instructions  | Optional. Do not link to the documentation as that is linked automatically. Do not include "basic" information like "Here you can set up X". |
+|       `data`       |    Field labels    | Keep succinct and consistent with other integrations whenever appropriate for the best user experience.                                      |
+| `data_description` | Field descriptions | Optional explanatory text to show below the field.                                                                                           |
+
+The field labels and descriptions are given as a dictionary with keys corresponding to your schema. Here is a simple example:
+
+```json
+{
+  "config": {
+    "step": {
+      "user": {
+          "title": "Add Group",
+          "description": "Some description",
+          "data": {
+              "entities": "Entities",
+          },
+          "data_description": {
+              "entities": "The entities to add to the group",
+          },
+      }
+    }
+  }
+}
+```
+
+#### Enabling Browser Autofill
+
+If your integration is collecting form data which can be automatically filled by browsers or password managers, such as login credentials or contact information, then you have 2 options to enable this feature.
```

Comment:
```suggestion
Suppose your integration collects form data that can be automatically filled by browsers or password managers, such as login credentials or contact information. In that case, you have two options to enable this feature.
```

Reply:
I agree the sentence was too long, but it should sound more imperative than this or even the way I wrote it.  Try the change I pushed.